sample-functions/AlpineFunction/Dockerfile.armhf

```diff
@@ -1,7 +1,8 @@
 FROM alpine:3.7
 
-ADD https://github.com/openfaas/faas/releases/download/0.9.0/fwatchdog-armhf /usr/bin/fwatchdog
-RUN chmod +x /usr/bin/fwatchdog
+RUN apk --no-cache add make curl \
```

Comment:
Why do we need `make`?

Reply:
My bad, it was just copy-pasted. I'll discard it from the files where it's not needed.